<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Router Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_statistics">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_title_connections_count">
            <property name="text">
             <string>Connections count:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_connections_count">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_active">
          <column>
           <property name="text">
            <string notr="true">ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connection time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_disconnect">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_disconnect_all">
            <property name="text">
             <string>Disconnect all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_refresh_active">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_proxy">
       <attribute name="title">
        <string>Proxy List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="button_add_proxy">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_modify_proxy">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/pencil.png</normaloff>:/img/pencil.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_proxy">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_proxy">
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Enabled</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_users">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="button_add_user">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_modify_user">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/pencil.png</normaloff>:/img/pencil.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_user">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>198</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_users">
          <column>
           <property name="text">
            <string>User name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Enabled</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_logging">
       <attribute name="title">
        <string>Logging</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_log_page">
            <property name="text">
             <string>Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combobox_log_page"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_prev_log_page">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/arrow-180.png</normaloff>:/img/arrow-180.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_next_log_page">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources/router_manager.qrc">
              <normaloff>:/img/arrow.png</normaloff>:/img/arrow.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_log">
          <column>
           <property name="text">
            <string>Time stamp</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Action</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/router_manager.qrc"/>
 </resources>
 <connections/>
</ui>
